app/drivers/tools/repair/java/AlphaRepair.py

```diff
@@ -0,0 +1,130 @@
+import os
+from os.path import join
+
+from app.drivers.tools.repair.AbstractRepairTool import AbstractRepairTool
+
+
+class AlphaRepair(AbstractRepairTool):
+    """
+    Requirements for this tool:
+    10 GB of VRAM, at most 7.0 CUDA (e.g. Nvidia V100) compute and 5 GB of RAM
+    """
+
+    def __init__(self):
+        self.name = os.path.basename(__file__)[:-3].lower()
+        super().__init__(self.name)
+        self.image_name = "thanhlecong/alpharepair:v1"
+        self.bug_name = ""
+
+    def run_repair(self, bug_info, repair_config_info):
+        super(AlphaRepair, self).run_repair(bug_info, repair_config_info)
+        """
+            self.dir_logs - directory to store logs
+            self.dir_setup - directory to access setup scripts
+            self.dir_expr - directory for experiment
+            self.dir_output - directory to store artifacts/output
+        """
+
+        timeout_h = str(repair_config_info[self.key_timeout])
+
+        if len(bug_info[self.key_fix_lines]) == 0:
+            self.error_exit("no line number to fix")
+
+        self.bug_name = bug_info[self.key_bug_id]
+        file = (
+            join(
+                bug_info[self.key_dir_source],
+                bug_info[self.key_fix_file].replace(".", "/"),
+            )
+            + ".java"
+        )
+    
+        repair_command = "bash -c 'export PATH=$PATH:/root/defects4j/framework/bin && timeout -k 5m {}h python3 main.py --task repair --bug_id {} --src_dir {} --buggy_file {} --buggy_loc {} --re_rank --beam_width 25 --top_n_patches 5000 --output_folder {}'".format(  # currently supporting only defects4j
```

Comment:
Can you change the top_n_patches to be a parameter? just so the value is not inlined

Reply:
Do you mean a parameter of Cerberus or a parameter of AlphaRepair?

Reply:
Sorry for the unclear statement - I am proposing of making the value `5000` passed as a parameter in the string.format call and have a variable containing the number `5000` so that later it can be changed either from the config info or manually by somebody else. Just a stylistic thing, nothing big

Reply:
ah I see, I will make the change.

Reply:
Done in the lastest commit.